docs/integration/social-networks.md

```diff
@@ -0,0 +1,31 @@
+# Social networks
+
+Social network use PHP backend library [Hybridauth](https://hybridauth.github.io/)
+
+## How to set it up?
+
+For every social network, you need `app_id` and `app_secret`. Below you have information on where and how to get them
```

Comment:
```suggestion
For every social network, you need `app_id` and `app_secret`. Below you have information on where and how to get them. The credentials then need to be set to the corresponding environment variables.
```

Reply:
fixed in  https://github.com/shopsys/shopsys/pull/3154/commits/57d32d7f080d7a42ea5f03bdb589c7daac49c5b3